src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -75,10 +75,6 @@ internal sealed class HttpConnectionPool : IDisposable
         private SemaphoreSlim? _http2ConnectionCreateLock;
         private byte[]? _http2AltSvcOriginUri;
         internal readonly byte[]? _http2EncodedAuthorityHostHeader;
-
```

Comment:
I think this is here for analyzers, not for direct functional impact. Unless needed for the Alt-Svc  I would probably not twist the changes together.

Reply:
For analyzers we have this one:
https://github.com/dotnet/runtime/blob/cc6537e4dddcea7886220ffc441641b97ed46837/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs#L282-L285